src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
```

Comment:
The elements `<ul>` ` <li>` ` <a>`   are part of the same `navigation` block. Please check your code and refactor according to this rule.
```suggestion
        <ul class="navigation__list">
          <li class="navigation__item">
            <a href="#" class="navigation__link is-active">Apple</a>
```